openlibrary/macros/LoanStatus.html

```diff
@@ -20,6 +20,7 @@
 
 $ user_loan = None
 $if user:
+    $ account = get_ol_account(ctx.user.key)
```

Comment:
Are we sure this should be getting the OL account? Shouldn't it be considering the OL and IA accounts?